io.github.erkin.ponysay.yaml

```diff
@@ -0,0 +1,14 @@
+app-id: io.github.erkin.ponysay
+runtime: org.freedesktop.Platform
+runtime-version: '19.08'
+sdk: org.freedesktop.Sdk
+command: /app/bin/ponysay
+modules:
+  - name: ponysay
+    buildsystem: simple
+    build-commands:
+      - ./setup.py --prefix /app --freedom=partial install
+    sources:
+      - type: git
+        url: https://github.com/erkin/ponysay.git
```

Comment:
this needs to be a specific commit or release, you shouldn't just point at master as the build can't be exactly reproduced then.

Reply:
Added latest tag and commit hash to yaml.